pages/linux/bpytop.md

```diff
@@ -0,0 +1,28 @@
+# bpytop
+
+> Display dynamic real-time information about running processes with graphs. Similar to `gtop` and `htop`.
+> More information <https://github.com/aristocratos/bpytop>.
+
+- Start bpytop:
+
+`bpytop`
+
+- Start in minimal mode without memory and networking boxes:
```

Comment:
I'm unsure of the actual behavior but to keep it stating the same as on the description, perhaps this will be better:
```suggestion
- Start in minimal mode without memory and networking graphs:
```
Also, if by "box" you don't mean "graph", I'm actually Ok on leaving it the way it is.

Reply:
![bpytop-screenshot](https://user-images.githubusercontent.com/37895518/91624375-ff2bf780-e96d-11ea-9e8b-3898a51cba8f.png) I do see where you are coming from, but this clearly states boxes. Thanks!

Reply:
Done!

Reply:
Nice. Thank you for your time :)

Reply:
No problem!